src/pages/Address/styled.tsx

```diff
@@ -5,313 +5,80 @@ export const AddressContentPanel = styled.div`
-  font-size: 50px;
-  color: rgb(20, 20, 20);
-  height: 70px;
-  text-align: center;
-
-  @media (max-width: 700px) {
-    margin-top: 30px;
-    height: 50px;
-    margin-bottom: 5px;
-    font-size: 26px;
-  }
-
-  &:after {
-    display: block;
-    content: '';
-    margin: 0 auto;
-    background: #3cc68a;
-    height: 4px;
-    width: 197px;
-
-    @media (max-width: 700px) {
-      width: 100px;
+  .address__pending_reward_help {
+    margin-left: 20px;
+    transform: translateY(2px);
+    position: relative;
+    > img {
+      width: 20px;
+      height: 20px;
     }
   }
 `
 
-export const AddressCommonContent = styled.div`
+export const AddressLockScriptPanel = styled.div`
   width: 100%;
-  overflow-x: auto;
-  padding: 72px 60px 56px 88px;
-  margin: 0 auto;
-  margin-top: 50px;
-  background: white;
-  border: 0px solid white;
-  border-radius: 6px;
-  box-shadow: 0px 5px 9px rgb(233, 233, 233);
-  display: flex;
-  flex-direction: column;
-
-  @media (max-width: 700px) {
-    width: 88%;
-    overflow-x: auto;
-    padding: 10px;
-    margin: 20px 6%;
-    background: white;
-    border: 0px solid white;
-    border-radius: 6px;
-    display: flex;
-    flex-direction: column;
+  .address__lock_script_title {
+    font-weight: 500;
   }
 `
 
-export const AddressTransactionsPanel = styled.div`
-  width: 100%;
-  overflow-x: auto;
-`
-
-export const AddressTransactionsItem = styled.div`
-  padding: 38px 83px 41px 83px;
-  margin: 0 auto;
-  margin-top: 10px;
-  background: white;
-  border: 0px solid white;
-  border-radius: 6px;
-  box-shadow: 0px 5px 9px rgb(233, 233, 233);
-  display: flex;
-  flex-direction: column;
-
-  .transaction__hash__panel {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-
-    .transaction_hash {
-      font-size: 16px;
-      color: rgb(75, 188, 142);
-    }
-
-    .transaction_block {
-      font-size: 16px;
-      color: rgb(75, 188, 142);
-    }
-  }
-
-  .transaction__separate {
-    width: 100%;
-    height: 1px;
-    margin-top: 30px;
-    background: rgb(233, 233, 233);
-  }
-
-  .transaction__input__output {
-    margin-top: 30px;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-
-    > img {
-      width: 68px;
-      height: 68px;
-    }
-  }
-`
-
-export const AddressScriptLabelPanel = styled.div`
+export const AddressLockScriptItemPanel = styled.div`
   display: flex;
   flex-direction: row;
   align-items: center;
-  height: 28px;
-
-  > img {
-    width: 20px;
-    height: 20px;
+  align-items: flex-start;
+  margin-top: 10px;
 
-    @media (max-width: 700px) {
-      width: 15px;
-      height: 15px;
-      display: none;
+  .address_lock_script__title {
+    > img {
+      width: 9px;
+      height: 9px;
     }
-  }
 
-  > span {
-    font-size: 18px;
-    font-weight: 450;
-    @media (max-width: 700px) {
+    > span {
+      margin-left: 10px;
+      font-family: Montserrat;
```

Comment:
Does `Montserrat ` be supported on linux, Windows and macOS ?

Reply:
macOS supports `Montserrat` font.

Reply:
If the system does not support `Montserrat `, the default font will be used.

Reply:
This font is not the same in Safari and Chrome.
I have modified the code. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/144/commits/74086fe1bc51690e6e16d15121df40490b1e45cf https://github.com/nervosnetwork/ckb-explorer-frontend/pull/144/commits/49a0197773816e01a8ed018b80c1759014951632


Can I use a local font file?

Reply:
No, font file will make js package more larger.

You can set more font families for other os which does not support `Montserrat `.